gitlab/base.py

```diff
@@ -51,6 +52,11 @@ class RESTObject(object):
     manager: "RESTManager"
 
     def __init__(self, manager: "RESTManager", attrs: Dict[str, Any]) -> None:
+        if not isinstance(attrs, dict):
+            raise GitlabParsingError(
+                "Attempted to initialize RESTObject with a non-dictionary value: "
+                "{!r}".format(attrs)
```

Comment:
Ok, maybe just an extra sentence for the user is all I can think of right now :)

```suggestion
                "Attempted to initialize RESTObject with a non-dictionary value: "
                "{!r}\n"
                "This likely indicates an incorrect or malformed server response.".format(attrs)
```

(black might disagree with this web IDE suggestion)

Reply:
Done